src/main/java/mysticalmechanics/util/Misc.java

```diff
@@ -8,24 +9,26 @@
 import net.minecraft.world.WorldServer;
 
 public class Misc {
-    static double epsilion = 0.00001;
+	static double epsilion = 0.00001;
 
-    public static boolean isRoughlyEqual(double a, double b) {
-        return a == b || Math.abs(a - b) < epsilion;
-    }
+	public static boolean isRoughlyEqual(double a, double b) {
+		return a == b || Math.abs(a - b) < epsilion;
+	}
 
-    public static void syncTE(TileEntity tile) {
-        World world = tile.getWorld();
-        if(world instanceof WorldServer) {
-            SPacketUpdateTileEntity packet = tile.getUpdatePacket();
-            if (packet != null) {
-                PlayerChunkMap chunkMap = ((WorldServer) world).getPlayerChunkMap();
-                int i = tile.getPos().getX() >> 4;
-                int j = tile.getPos().getZ() >> 4;
-                PlayerChunkMapEntry entry = chunkMap.getEntry(i, j);
-                if(entry != null)
-                    entry.sendPacket(packet);
-            }
-        }
-    }
+	public static void syncTE(TileEntity tile) {
+		World world = tile.getWorld();
+		if (world instanceof WorldServer) {
+			PlayerChunkMap chunkMap = ((WorldServer) world).getPlayerChunkMap();
+			SPacketUpdateTileEntity packet = tile.getUpdatePacket();
+			if (packet != null) {
+				int i = tile.getPos().getX() >> 4;
+				int j = tile.getPos().getZ() >> 4;
+				PlayerChunkMapEntry entry = chunkMap.getEntry(i, j);
+				if (entry != null)
+					entry.sendPacket(new SPacketBlockChange(chunkMap.getWorldServer(),tile.getPos()));					 
```

Comment:
you'll want to put this stuff in {} or it will potentially cause an NPE on the second packet

Reply:
wow good catch i totally failed that......